packages/google_maps_flutter/google_maps_flutter_android/android/src/main/java/io/flutter/plugins/googlemaps/CozyMarkerBuilder.java

```diff
@@ -72,22 +73,16 @@ private int getMarkerSize() {
             return Math.max(proportionalMarkerSize, minMarkerSize);
     }
 
-    private Bitmap getEmptyClusterBitmap(int size) {
```

Comment:
i've removed the method to create a empty round marker since storing a bitmap didn't made any performant changes, and storing a blank bitmap in memory don't seem as necessary now that we're caching the whole marker